coverity.rst

```diff
@@ -143,10 +143,10 @@ Dakshesh Vyas <scan-admin@coverity.com>
 
 .. seealso::
 
-   `Coverity Scan FAQ <http://scan.coverity.com/faq/>`_
+   `Coverity Scan FAQ <https://scan.coverity.com/faq/>`_
 
 
-.. _Coverity Scan: http://scan.coverity.com/
+.. _Coverity Scan: https://scan.coverity.com/
 
 .. _Coverity Connect: http://scan5.coverity.com:8080/
```

Comment:
Please change this to https://scan.coverity.com/projects/python

Reply:
@encukou Gotcha! :)